src/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27"
+      >
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="stars_reviews"> Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <span class="price-word">Price:</span>
+        <span class="price-amount">$2,199</span>
+      </div>
+
+      <a href="#" data-qa="hover">
+        <button type="button" class="card__button">Buy</button>
+      </a>
```

Comment:
```suggestion
      <a href="#" class="card__button" data-qa="hover" >
        Buy
      </a>
```
you may get rid of the `button` tag
write it in this way (add some additional styles if it's needed)